packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -357,7 +358,55 @@ $ms-Persona-presenceSizeXl: 28px;
   .presence {
     height: $ms-Persona-presenceSizeXxs;
     width: $ms-Persona-presenceSizeXxs;
-    border: 1px solid #ffffff;
+    border: 4px solid #ffffff;
```

Comment:
Yes they were missing - and I followed Angela's red lines and other examples in the file - please let me know if something isn't to spec and / or should be removed.